docs/content/latest/explore/ysql-language-features/tablespaces.md

```diff
@@ -0,0 +1,100 @@
+
+For YugabyteDB clusters, however, location does not pertain to disk locations. For a cloud-native distributed database, location pertains to the cloud, region, and zone where the data is supposed to be. Therefore, although YSQL tablespaces are built on PostgreSQL tablespaces that allow you to specify placement of data at a table level, this placement information in YSQL defines the number of replicas for a table and index, as well as how they can be distributed across a set of cloud, regions, and zones. 
+
+## Defining a Tablespace
+
+You can define a tablespace using the following syntax:
+
+```sql
+CREATE TABLESPACE tablespace_name 
+  OWNER username
+  WITH (replica_placement = placement_policy_json);
+```
+
+In the preceding syntax:
+
+- *tablespace_name* represents the name of the tablespace to be created. 
+- *username* represents the name of the user who will own the tablespace, with the name of the user executing the command being the default. Note that only superusers can create tablespaces and grant their ownership to other types of users. 
+- *placement_policy_json* represents a JSON string that specifies the placement policy for this tablespace. The JSON structure contains the following two fields: 
+  - `num_replicas` defines the overall replication factor.
+  - `placement_blocks` is an array of tuples, with each tuple containing the keys `<”cloud”, “region”, “zone”, “min_num_replicas”>` whose values define a placement block. Typically, the sum of `min_num_replicas` across all placement blocks is expected to be equal to `num_replicas`. The aggregate of `min_num_replicas` can be lesser than `num_replicas`, in which case the extra replicas are placed at the YB-Load balancer’s discretion.
+
+The following example shows how to create a tablespace:
+
+```sql
+CREATE TABLESPACE us_west_tablespace 
+WITH (replica_placement='{"num_replicas": 3, "placement_blocks":
+[{"cloud":"aws","region":"us-west","zone":"us-west-1a","min_num_replicas":1},
+{"cloud":"aws","region":"us-west","zone":"us-west-1b","min_num_replicas":1},
+{"cloud":"aws","region":"us-west","zone":"us-west-1c","min_num_replicas":1}]}');
+```
+
+If you create a table and use the preceding tablespace, each tablet of the table will have three replicas, as per the `num_replicas` value. Since `placement_blocks` contains three placement blocks where `min_num_replicas` is set to 1, YB-Load balancer will ensure that `aws.us-west.us-west-1a`, `aws.us-west.us-west-1b`, and `aws.us-west.us-west-1c` will each have one replica.
+
+The following example demonstrates how to create a tablespace in which the value of `min_num_replicas` does not correspond to `num_replicas`:
+
+```sql
+CREATE TABLESPACE us_east_tablespace 
```

Comment:
Please add the notice output within the same sql block - I think that is what Karthik mentioned later in the slack thread.

Reply:
I'd rather not have this Notice in the same code block as the statement. People use the "copy" button and then paste in their console whatever got copied, then they hit return. 
(That's why I separated these two in the first place)
I suggest we either have a separate code block for the Notice or not have Notice all, just the description.